conf/modules/mutect2.config

```diff
@@ -14,110 +14,137 @@
-        ]
-    }
+        withName: 'GATHERPILEUPSUMMARIES' {
+            ext.prefix       = { "${meta.id}.mutect2" }
+            publishDir       = [
+                mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/mutect2/${meta.id}/" },
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            ]
+        }
 
-    withName: 'LEARNREADORIENTATIONMODEL' {
-        ext.prefix       = { "${meta.id}.mutect2.artifactprior" }
-        publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/mutect2/${meta.id}/" },
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-        ]
-    }
+        withName: 'GATHERPILEUPSUMMARIES_.*' {
+            ext.prefix       = { "${meta.id}.mutect2" }
+            publishDir       = [
+                mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/mutect2/${meta.tumor_id}_vs_${meta.normal_id}/" },
+                saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            ]
+        }
 
-    withName: 'MERGEMUTECTSTATS' {
-        ext.prefix       = { "${meta.id}.mutect2" }
-        publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/mutect2/${meta.id}/" },
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
-        ]
-    }
+        withName: 'GETPILEUPSUMMARIES.*' {
+            ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}.mutect2" : "${meta.id}.mutect2.${intervals.simpleName}" }
+            publishDir        = [
+                mode: params.publish_dir_mode,
+                path: { "${params.outdir}/variant_calling/" },
+                pattern: "*.table",
+                saveAs: { meta.num_intervals > 1 ? null : "mutect2/${meta.id}/${it}" }
+            ]
+        }
 
-    withName: 'GATK4_MUTECT2' {
-        ext.prefix       = { meta.num_intervals <= 1 ? "${meta.id}.mutect2" : "${meta.id}.mutect2.${intervals.simpleName}" }
-        ext.when         = { params.tools && params.tools.split(',').contains('mutect2') }
-        ext.args         = { params.ignore_soft_clipped_bases ? "--dont-use-soft-clipped-bases true --f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz" : "--f1r2-tar-gz ${task.ext.prefix}.f1r2.tar.gz" }
-        publishDir       = [
-            mode: params.publish_dir_mode,
-            path: { "${params.outdir}/variant_calling/" },
-            pattern: "*{vcf.gz,vcf.gz.tbi,stats}",
-            saveAs: { meta.num_intervals > 1 ? null : "mutect2/${meta.id}/${it}" }
-        ]
-    }
+        withName: 'GETPILEUPSUMMARIES_.*' {
+            ext.prefix        = { meta.num_intervals <= 1 ? "${meta.id}.mutect2" : "${meta.id}.mutect2.${intervals.simpleName}" }
```

Comment:
I think this line can be removed, already taken care of in the more general one above
```suggestion
```